src/Products/urban/content/licence/CODT_UniqueBorderingLicence.py

```
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+
+from AccessControl import ClassSecurityInfo
+from Products.Archetypes.atapi import *
```

Ruff (0.15.2)

[error] `from Products.Archetypes.atapi import *` used; unable to detect undefined names

(F403)